neos/neos-ui
# FEATURE: Remember active tab in inspector

With this change the sidebar will remember
which inspector tab was selected last and
restore the correct tab when switching between
nodes. Even if the order of the tabs is different.

**What I did**

Instead of storing the active tabs index I store its id which is unique.

**How I did it**

Just store the different property.

**How to verify it**

Select the document node and the SEO tab, then switch between content nodes and the document node and see that the tab will be restored.
